compiler/packages/babel-plugin-react-compiler/src/Entrypoint/Options.ts

```diff
@@ -102,13 +102,7 @@ export type PluginOptions = Partial<{
 
   panicThreshold: PanicThresholdOptions;
 
-  /*
-   * When enabled, Forget will continue statically analyzing and linting code, but skip over codegen
-   * passes.
-   *
-   * Defaults to false
-   */
-  noEmit: boolean;
```

Comment:
should we deprecate rather than remove? while we haven't documented this option in the docs, it might be debatable if this would be considered a breaking change or not